Stabilization changes

Keshash opened the pull request:
## Additions

Describe new functionality added by your code, e.g.

- Stabilizing now creates trash. 5 textures for industrial and 5 for herbal. Industrial trash is uglier, cleaner, lasts longer. Herbal trash is dirtier, disappears quicker. Glitterworld meds don't create trash because they have such good packaging and only crazy people spend glitter to stabilize. All other meds don't create trash.
- Stabilization icon replaced with a higher resolution one. Made icon change color (from brown - bad, to white - good) in accordance with stabil. quality.  
- Added text that shows how long stabil. will last and how much it reduces bleeding rate to wound inspection tool. Made Eng+Rus translations to this text.
- This functionality can be attached to modded medicine by patching in a `MedicineFilthExtension` modExtension with following fields: `filthDefName` - what filth will spawns (by default, none will), `filthSpawnChance` - chance that any filth will spawn at all, `filthSpawnQuantity` - range of how much separate filth objects will spawn if possible.

## Reasoning

Why did you choose to implement things this way, e.g.
- More informative tooltip makes it easier to understand how stabilization works
- Aesthetics

## Alternatives

Describe alternative implementations you have considered, e.g.
- Create graphics for glitterworld meds trash
- Change colors of the icon or the icon itself to something different.

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [ ] Playtested a colony (specify how long)

N7Huntsman reviewed (commented):
I'm onboard with the higher-res texture and added tool-tips.

For the filth; given the possibility of other modded medicines being used for stabilization, would it make sense to make this a bit more like the cartridge code where we can dictate the filth created in xml? As it stands, it looks fairly fixed.

Safairette commented on Source/CombatExtended/CombatExtended/Comps/HediffComp_Stabilize.cs:
```diff
@@ -109,13 +106,33 @@ public override TextureAndColor CompStateIcon
-                {
-                    return new TextureAndColor(StabilizedIcon, Color.white);
+                { //Change icon color with stabilization quality: brown - bad, yellow - good. 
+                    Color color = Color.Lerp(HediffComp_TendDuration.UntendedColor, new ColorInt(255, 212, 136).ToColor, Mathf.Clamp01(1 - bleedModifier));
```
Any reason why we aren't using the vanilla white for 100% quality stabilisation?

The different icon should be clear enough as is to tell them apart and that way we'll be keeping the UI more consistent by sticking to the existing color scheme

Keshash commented:
* Fixed phrasing in translations
* Herbal meds now spawn 1-3 trash piles.
* Now neighbouring 8 tiles can be used for spawning instead of only the one that medic stands on
* Added a modExtension to attach this function to modded medicine
* Changed good tends color to white, made icon a little smaller.

N7Huntsman reviewed (approved):
Worked in testing.
